File: AR-VR/saurabhcr007_saurabhchaudhary_1923csi1081_3/GettingStarted/IsANumberPrime.java
import java.util.*;

public class IsANumberPrime {
    public static void main(String[] args)  {
        Scanner scn = new Scanner(System.in);
      int t=scn.nextInt();
      int count=0;
      for(int i=0;i<t;i++){count=0;
          int n=scn.nextInt();
          for(int j=2;j<n;j++){
              if(n%j==0){
                  count++;
              }
              if(count>0){
                  break;
              }
          }
          if(count==0){
              System.out.println("prime");
          }
          else{
              System.out.println("not prime");
          }
      }
    }
}    

File: AR-VR/vanshverma721_VanshVerma_2024csit1153_2/GettingStarted/IsANumberPrime.cpp
#include <iostream>
using namespace std;
int main()
{
   int t,n,i,j;
   scanf("%d",&t);
   for(i=0;i<t;i++)
   {
       scanf("%d",&n);
       
       int count=0;
       for(j=2;j*j<=n;j++)
       {
           if(n%j==0)
           {
               count++;
               break;
           }
       }
       
       if(count==0)
       {
           printf("prime\n");
       }
       else
       {
           printf("not prime\n");
       }
       
   }

return 0;
    
}